spec/std/isa/csr/mstatus.yaml

```
@@ -65,6 +65,25 @@ fields:
     definedBy: Smdbltrp
     type: RW-H
     reset_value: UNDEFINED_LEGAL
+  SDT:
+    location: 41
```

Am I mistaken (again)? I'm seeing these fields at position 24. (I see position 41 as "MPELP".)
```suggestion
    location: 24
```